src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/UnusedTagsPassTest.php

```diff
@@ -31,4 +31,23 @@ public function testProcess()
 
         $this->assertSame([sprintf('%s: Tag "kenrel.event_subscriber" was defined on service(s) "foo", "bar", but was never used. Did you mean "kernel.event_subscriber"?', UnusedTagsPass::class)], $container->getCompiler()->getLog());
     }
+
+    public function testMissingWhitelistTags()
+    {
+        $this->assertSame(UnusedTagsPassUtils::getDefinedTags(), $this->getWhitelistTags(), 'The src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php file must be updated; run src/Symfony/Bundle/FrameworkBundle/Resources/bin/check-unused-tags-whitelist.php.');
```

Comment:
We should skip this test for deps=* tests. I'd suggest doing so this way:
`if (!defined('PHPUNIT_COMPOSER_INSTALL') || dirname(PHPUNIT_COMPOSER_INSTALL, 2) !== dirname(__DIR__, 5)) // skip`
(if I counted correctly for the 2nd args)

Reply:
hum this check doesn't work :)
Another blind try:
`dirname((new ReflectionClass(CompilerPassInterface::class))->getFileName(), 3) === dirname(__DIR__, 4)`